<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>659</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>949</width>
    <height>659</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>949</width>
    <height>659</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: rgb(65, 87, 66); 
}</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyOpenGLWidget" name="GLWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_open_file">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     background-color: white;
     border-style: outset;
     border-radius: 7px;
     border-color: beige;
     font: bold 14px;
     padding: 6px;
	 width: 50px; 
	color:rgb(63, 82, 76);
 }
 QPushButton:pressed {
     background-color: rgb(215,205, 196);
     border-style: inset;
 }
</string>
    </property>
    <property name="text">
     <string>Открыть файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>661</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_screen">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     background-color: white;
     border-style: outset;
     border-radius: 7px;
     border-color: beige;
     font: bold 14px;
     padding: 6px;
	 width: 50px; 
	color:rgb(63, 82, 76);
 }
 QPushButton:pressed {
     background-color: rgb(215,205, 196);
     border-style: inset;
 }
</string>
    </property>
    <property name="text">
     <string>Скрин</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gif">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     background-color: white;
     border-style: outset;
     border-radius: 7px;
     border-color: beige;
     font: bold 14px;
     padding: 6px;
	 width: 50px; 
	color:rgb(63, 82, 76);
 }
 QPushButton:pressed {
     background-color: rgb(215,205, 196);
     border-style: inset;
 }
</string>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_X">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_X">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>130</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Перемещение</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Поворот</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>120</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>160</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>160</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>120</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>540</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Масштаб</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="scale">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>540</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>0</number>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>10.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>230</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Проекция:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>290</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Тип линии</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>340</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Толщина ребер</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="size_lines">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>340</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Размер вершин</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="size_of_point">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>370</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>480</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Способ отображения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>400</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
     font: bold 14px;
	color:white;
}</string>
    </property>
    <property name="text">
     <string>Цвет</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>420</y>
      <width>251</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="color_back">
       <property name="styleSheet">
        <string notr="true"> QPushButton {
     background-color: white;
     border-style: outset;
     border-radius: 7px;
     border-color: beige;
     font: bold 14px;
     padding: 6px;
	 width: 50px; 
	color:rgb(63, 82, 76);
 }
 QPushButton:pressed {
     background-color: rgb(215,205, 196);
     border-style: inset;
 }
</string>
       </property>
       <property name="text">
        <string>фон</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="color_points">
       <property name="styleSheet">
        <string notr="true"> QPushButton {
     background-color: white;
     border-style: outset;
     border-radius: 7px;
     border-color: beige;
     font: bold 14px;
     padding: 6px;
	 width: 50px; 
	color:rgb(63, 82, 76);
 }
 QPushButton:pressed {
     background-color: rgb(215,205, 196);
     border-style: inset;
 }
</string>
       </property>
       <property name="text">
        <string>вершины</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="color_line">
       <property name="styleSheet">
        <string notr="true"> QPushButton {
     background-color: white;
     border-style: outset;
     border-radius: 7px;
     border-color: beige;
     font: bold 14px;
     padding: 6px;
	 width: 60px; 
	color:rgb(63, 82, 76);
 }
 QPushButton:pressed {
     background-color: rgb(215,205, 196);
     border-style: inset;
 }
</string>
       </property>
       <property name="text">
        <string>рёбра</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="save_settings">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>570</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     background-color: white;
     border-style: outset;
     border-radius: 7px;
     border-color: beige;
     font: bold 14px;
     padding: 6px;
	 width: 50px; 
	color:rgb(63, 82, 76);
 }
 QPushButton:pressed {
     background-color: rgb(215,205, 196);
     border-style: inset;
 }
</string>
    </property>
    <property name="text">
     <string>Сохранить настройки</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_Y">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="move_Z">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>190</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_Y">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="rotate_Z">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>190</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="use_settings">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>610</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> QPushButton {
     background-color: white;
     border-style: outset;
     border-radius: 7px;
     border-color: beige;
     font: bold 14px;
     padding: 6px;
	 width: 50px; 
	color:rgb(63, 82, 76);
 }
 QPushButton:pressed {
     background-color: rgb(215,205, 196);
     border-style: inset;
 }
</string>
    </property>
    <property name="text">
     <string>Применить настройки</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>220</y>
      <width>126</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="projection_parallel">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
     font: bold 14px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>параллельная</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="projection_central">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
     font: bold 14px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>центральная</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>280</y>
      <width>109</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="line_solid">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
     font: bold 14px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>сплошная</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="line_dots">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
     font: bold 14px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>пунктирная</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="round_point">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
     font: bold 14px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>круг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="no_point">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
     font: bold 14px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>нет</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="square_point">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
     font: bold 14px;
	color:white;
}</string>
       </property>
       <property name="text">
        <string>квадрат</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>myopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
